src/containers/LearnerDashboardHeader/ConfirmEmailBanner/index.test.jsx

```diff
@@ -20,16 +25,52 @@ const hookProps = {
 };
 
 describe('ConfirmEmailBanner', () => {
-  describe('snapshot', () => {
-    test('do not show on already verified', () => {
+  describe('render', () => {
+    it('do not show on already verified', () => {
       hooks.mockReturnValueOnce({ ...hookProps, isNeeded: false });
-      const el = shallow(<ConfirmEmailBanner />);
-      expect(el.snapshot).toMatchSnapshot();
+      render(<IntlProvider locale="en"><ConfirmEmailBanner /></IntlProvider>);
+      const banner = screen.queryByRole('alert');
+      expect(banner).toBeNull();
     });
-    test('Show on unverified', () => {
-      hooks.mockReturnValueOnce({ ...hookProps });
-      const el = shallow(<ConfirmEmailBanner />);
-      expect(el.snapshot).toMatchSnapshot();
+    describe('On unverified', () => {
+      it('show banner', () => {
+        hooks.mockReturnValueOnce({ ...hookProps });
+        render(<IntlProvider locale="en"><ConfirmEmailBanner /></IntlProvider>);
+        const banner = screen.getByRole('alert');
+        expect(banner).toContainHTML('Remember to confirm');
+      });
```

Comment:
I believe we can de-duplicate some of this code by using `beforeAll` and mocking the `hooks` for all tests so that the `hooks.mockReturnValueOnce({ ...hookProps });` mock doesn't need to be repeated for each test block.

```suggestion
    describe('On unverified', () => {
      beforeAll(() => {
        hooks.mockReturnValue({ ...hookProps });
      });
      it('show banner', () => {
        render(<IntlProvider locale="en"><ConfirmEmailBanner /></IntlProvider>);
        const banner = screen.getByRole('alert');
        expect(banner).toContainHTML('Remember to confirm');
      });
```

Reply:
done 👍